contracts/governance/Governor.sol

```diff
@@ -837,6 +837,13 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
      */
     function quorum(uint256 timepoint) public view virtual returns (uint256);
 
+    /**
+     * @inheritdoc IGovernor
+     */
+    function proposalVotes(
+        uint256 proposalId
+    ) public view virtual returns (uint256 againstVotes, uint256 forVotes, uint256 abstainVotes);
+
```

Comment:
IMO we should not add this here. Not all counting systems have will implement this function (there are "grade-based" module that don't use for/against/abstain).

IMO its ok for the super quorum module to not work with them, and to only support counting module that expose that function

Reply:
Created a new `IGovernorCounting` interface and moved this function in it.